File: JavaScript/LeetCode_1470.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function(nums, n) {
       var x_list=[];
   var y_list=[];
   var add_list=[];
   for (var i=0;i<n;i++){
    x_list.push(nums[i]);
   } 
   for (var ii=n;ii<n*2;ii++){
    y_list.push(nums[ii]);
   } 
   for (var add=0;add<n;add++){
    add_list.push(x_list[add]);
    add_list.push(y_list[add]);
   }
   return add_list
};

File: Python/LeetCode_1470.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        x_list=[]
        y_list=[]
        add_list=[]
        for i in range(n):
            x_list.append(nums[i])
        for ii in range(n,n*2):
            y_list.append(nums[ii])
            
        for add in range(n):
            add_list.append(x_list[add])
            add_list.append(y_list[add])
        return add_list